packages/cw3/src/msg.rs

```diff
@@ -31,8 +30,8 @@ where
     },
 }
 
-#[derive(Serialize, Deserialize, Clone, Copy, PartialEq, Eq, JsonSchema, Debug)]
-#[serde(rename_all = "lowercase")]
+#[cw_serde]
+#[derive(Copy, Eq)]
```

Comment:
cw_serde have `rename_all = "snake_case"`. It shouldn't have any difference from lowercase for those variants